examples/run_quickstart_new_config.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import annotations
+
+import os
+from typing import Any
+
+import pandas as pd
+
+from tabrepo import EvaluationRepository, EvaluationRepositoryCollection, Evaluator
+from tabrepo.benchmark.experiment import AGModelBagExperiment, Experiment, ExperimentBatchRunner
+
+
+if __name__ == '__main__':
+    # Load Context
+    context_name = "D244_F3_C1530_10"  # 10 smallest datasets. To run larger, set to "D244_F3_C1530_200"
+    expname = "./experiments/quickstart_new_config"  # folder location to save all experiment artifacts
```

Comment:
Would be nice to avoid a relative location:
```suggestion
    expname = "Path(__file__).parent / "experiments" / "quickstart_new_config"  # folder location to save all experiment artifacts
```

Reply:
There was a typo in the code suggestion with extra `"`, but thanks for mentioning. I updated the code to address.